static/app/components/feedback/list/issueTrackingSignals.tsx

```diff
@@ -27,6 +27,20 @@ function filterLinkedPlugins(actions: ExternalIssueComponent[]) {
   return plugins.concat(nonPlugins);
 }
 
+function getPluginNames(pluginIssue) {
+  return {
+    name: pluginIssue.props.plugin.name ?? '',
```

Comment:
Should these say `?? t('Unknown')` or something like that?

Reply:
i feel like i'd rather just show 'linked issue' rather than 'linked unknown issue' wdyt? the empty icon name will map to a default icon

Reply:
Ah, that's fair.